tower_eval/cli.py

```diff
@@ -20,18 +20,18 @@
 import numpy as np
 from loguru import logger
 
-from tower_eval.models import available_models
```

Comment:
by removing this line we won't be able to provide the user with the list of supported models, right? Don't we want to keep this?

Reply:
well, I think we don't need this. so, you can ignore this comment.